maps/maps.go

```diff
@@ -5,8 +5,10 @@
 // Package maps contains the generic functions for maps.
 package maps
 
-// NOTE(DmitriyMV): I tried to implement this generic functions to be as perfomant as possible.
-// However, I couldn't find a way to do it, since Go (1.18 at the time of writing) cannot inline closures if (generic)
+import "maps"
+
+// NOTE(DmitriyMV): I tried to implement this generic functions to be as performant as possible.
+// However, I couldn't find any way to do it, since Go (1.18 at the time of writing) cannot inline closures if (generic)
```

Comment:
does this matter anymore? since we're already in 1.21?

Reply:
Inliner didn't got any smarter. They plan to do something in 1.22, but for now I think we should keep this comment.